### PR TITLE
Some minor edits to the website

### PR DESCRIPTION
**Pull request by heipei**
See description of patches.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ restic
...
+ * *Secure:* Restic uses cryptography to guarantee confidentiality and integrity of your data. The location the backup data is stored is assumed not to be a trusted environment (e.g. a shared space where others like system administrators are able to access your backups). Restic is built to secure your data against such attackers.
 
- * *Efficient*: With the growth of data, additional snapshots should only take the storage of the actual increment. Even more, duplicate data should be de-duplicated before it is actually written to the storage backend to save precious backup space.
+ * *Efficient:* With the growth of data, additional snapshots should only take the storage of the actual increment. Even more, duplicate data should be de-duplicated before it is actually written to the storage backend to save precious backup space.
 
- * *Free*: restic is free software and licensed under the [BSD 2-Clause License](https://github.com/restic/restic/blob/master/LICENSE)
+ * *Free:* restic is free software and licensed under the [BSD 2-Clause License](https://github.com/restic/restic/blob/master/LICENSE). restic is actively developed on [GitHub](https://github.com/restic/restic/).
```
Comment (fw42) [inline index.md:20]: sounds weird to start two sentences in a row with the same word.. just make it one sentence with an "and"?

File: index.md [hunk 2]
```diff
@@ -60,50 +62,60 @@ First, we need to create a "repository". This is the place where your backups wi
...
 
-    $ restic init --repository /tmp/backup init
-    enter password for new backend:
-    enter password again:
-    created restic backend 085b3c76b9 at /tmp/backup
-    Please note that knowledge of your password is required to access the repository.
-    Losing your password means that your data is irrecoverably lost.
+{% highlight console %}
+$ restic init --repository /tmp/backup init
+ enter password for new backend:
```
Comment (fw42) [inline index.md:67]: the indentation changed (there is a space now that wasn't there before). intentional?

File: index.md [hunk 3]
```diff
@@ -60,50 +62,60 @@ First, we need to create a "repository". This is the place where your backups wi
...
-    $ restic -r /tmp/backup backup ~/work
-    enter password for repository:
-    scan [/home/user/work]
-    scanned 764 directories, 1816 files in 0:00
-    [0:29] 100.00%  54.732 MiB/s  1.582 GiB / 1.582 GiB  2580 / 2580 items  0 errors  ETA 0:00
-    duration: 0:29, 54.47MiB/s
-    snapshot 40dc1520 saved
+{% highlight console %}
+$ restic -r /tmp/backup backup ~/work
+ enter password for repository:
```
Comment (fw42) [inline index.md:80]: same here re: space

**Comment by fd0** [comment]
I've fixed the additional space and the wording.

Nice work, thanks for your contribution!

**Pull request merged**